spec/features/training_tool_spec.rb

```diff
@@ -70,6 +70,7 @@
       login_as(user, scope: :user)
     end
 
+
```

Comment:
Looks like you removed the spec you had added?

This definitely needs a spec or two to exercise the feature.

Reply:
Was getting some errors with the tests, i am rewriting them.